frontend/packages/dev-console/src/components/pipelines/__tests__/Pipeline.spec.tsx

```diff
@@ -18,11 +19,29 @@ const pipelinePageProps = {
   },
 };
 
-const pipelineWrapper = shallow(<PipelinesPage {...pipelinePageProps} />);
+const pipelinePagePropsWNS = {
+  history: null,
+  location: null,
+  match: {
+    isExact: true,
+    path: `/k8s/ns/:ns/${referenceForModel(PipelineModel)}`,
+    url: 'k8s/ns/my-project/tekton.dev~v1alpha1~Pipeline',
+    params: {
+      ns: 'all-projects',
+    },
+  },
+};
 
 describe('Pipeline List', () => {
+  const pipelineWrapperNS = shallow(<PipelinesPage {...pipelinePageProps} />);
   it('Renders a list', () => {
-    expect(pipelineWrapper.exists()).toBe(true);
-    expect(pipelineWrapper.find(ListPage).exists());
+    expect(pipelineWrapperNS.exists()).toBe(true);
```

Comment:
you can use `toBeTruthy()`

Reply:
Could be either way. `toBe(true)` is certainly the more precise way to go and often what we want.
truthy: `!!value`
toBe(true): `true === value`

Personally i'd use `toBe(true)` because we know it's of type boolean.

Reply:
sticking to tobe(true)